cpg-core/src/main/java/de/fraunhofer/aisec/cpg/passes/scopes/ScopeManager.kt

```diff
@@ -465,9 +465,9 @@ class ScopeManager : ScopeProvider {
             }
             (scope as Breakable).addBreakStatement(breakStatement)
         } else {
-            val labelStatement = getLabelStatement(breakStatement.label)
+            val labelStatement = getLabelStatement(breakStatement.label!!)
```

Comment:
@KuechA please avoid !!, we cannot guarantee that all of these nodes are not-null

Reply:
I added two more checks about the nullable `subStatement`. We check that the label is null in the if-branch, so it should be fine with `!!` here.